SCons/Script/SConsOptions.py

```diff
@@ -40,7 +40,7 @@
 
 diskcheck_all = SCons.Node.FS.diskcheck_types()
 
-experimental_features = {'warp_speed', 'transporter', 'ninja', 'tm_v2'}
+experimental_features = {'warp_speed', 'transporter', 'ninja', 'legacysched'}
```

Comment:
how about legacy_sched (ala warp_speed).. easier to read/type correctly

Reply:
Done